hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/AbstractAsyncSocket.java

```diff
@@ -72,18 +72,18 @@ public final void setCloseListener(CloseListener listener, Executor executor) {
         checkNotNull(listener, "listener");
 
         // Using lock to make sure that there is a matching listener/executor.
-        boolean closeListenerChecked;
```

Comment:
I do not like this naming. Local is often used in different context, so I find this misleading.

Reply:
What do you recommend? `closeListenerChecked0` for example?

Reply:
'thisClosedListener' sounds like the least sucky version.